src/cascadia/TerminalApp/AppKeyBindingsSerialization.cpp

```diff
@@ -86,32 +86,42 @@ static const std::map<std::wstring_view, ShortcutAction> commandNames {
 };
 
 // Function Description:
-// - Small helper to insert a given KeyChord, ShortcutAction pair into the
-//   given json array
+// - Small helper to create a json value serialization of a single
+//   KeyBinding->Action maping. The created object is of schema:
+//   {
+//      keys:[String],
+//      command:String
+//   }
 // Arguments:
-// - bindingsArray: The JsonArray to insert the object into.
-// - chord: The KeyChord to serailize and place in the json array
+// - chord: The KeyChord to serialize
 // - actionName: the name of the ShortcutAction to use with this KeyChord
-static void _AddShortcutToJsonArray2(Json::Value& bindingsArray,
-                                     const KeyChord& chord,
-                                     const std::wstring_view& actionName)
+// Return Value:
+// - a Json::Value which is an equivalent serialization of this object.
+static Json::Value _ShortcutAsJsonObject(const KeyChord& chord,
+                                         const std::string_view& actionName)

```

Comment:
nit: string views don't need to be const references since they're so small/cheap